conf/modules.config

```diff
@@ -127,31 +127,33 @@ if(params.aligner == "cellrangerarc") {
     }
 }
 
-if (params.aligner == "alevin") {
+if (params.aligner == "simpleaf" || params.aligner == "alevin") {
```

Comment:
since in the `nextflow_schema.json` the aligner parameter has an `enum` statement, the workflow will never have `params.aligner == 'alevin'`, thus, I think this is not needed.

Reply:
Since alevin is not in the schema. I believe the pipeline will not allow. Can you give it a check trying to run with aligner alevin?